common/scala/build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
 
```

Comment:
From the above, spray-caching hasn't been ported to Akka yet. There is an issue open for it here: https://github.com/akka/akka-http/issues/213. Will look at replacing spray-client.